### PR TITLE
DataBase Week-1 Homework

### PR DESCRIPTION
**Pull request by myselfankitaa**

### CODE REVIEW

File: databases/week1/week1.sql [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+insert into task (id, title, description, created, updated, due_date, status_id, user_id) values (33, 'Hang up paintings in living room', NULL, '2017-09-10 05:36:11', '2017-10-09 17:40:42', null, 3, 4);
+insert into task (id, title, description, created, updated, due_date, status_id, user_id) values (34, 'Buy plane ticket to Auckland', 'Check prices online first!', '2017-09-05 09:07:22', '2017-10-15 09:36:06', '2017-12-07 11:10:05', 1, 9);
+insert into task (id, title, description, created, updated, due_date, status_id, user_id) values (35, 'Learn about NoSQL databases', 'MongoDB, CouchDB, etc.', '2017-10-20 01:41:53', '2017-10-04 07:19:56', '2017-12-23 10:13:42', 2, NULL);
+
+
+select count(*) from task
+
+select count(*) from task where due_date is NULL
+
+select * from task inner join status on task.status_id = status.id
```
Comment (NatsGt) [inline databases/week1/week1.sql]: By just writing JOIN, SQL uses a default inner join, just so you know.

File: databases/week1/week1.sql [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+WHERE status.name = "Done"
+
+select * from task left join status on task.status_id = status.id
+WHERE status.name != "Done"
```
Comment (NatsGt) [inline databases/week1/week1.sql:100]: A Join is enough, dont need to do a left join here.

**Review by NatsGt — COMMENTED** [review]
Very good job. Just so you know, you can just write JOIN each time you need a inner join, SQL has it as default.

File: databases/week1/week1.sql (hunk 1 above)
Comment (myselfankitaa) [reply databases/week1/week1.sql]: ok Noted

File: databases/week1/week1.sql (hunk 2 above)
Comment (myselfankitaa) [reply databases/week1/week1.sql:100]: ok noted

**Pull request merged**